src/pool/staging_accounts.json

```diff
@@ -1,32 +1,12 @@
 [
-  "0x6f644133991e779509cb319fd4c9416f949950f07a8bc3cb10916966ec1dca",
-  "0xc6d61dd26ec6175554977f61198b28f58a7c5f46228b7fb7b44f6f619d800",
-  "0x6faca0d5ce2437933257a91d95536c7cf3bfa12808f314741efd104e9a0c2d9",
-  "0x2cced3d1628b867e1b69216df87d869e7f5c8dd066a9114d44e2d1af3fbf2c4",
-  "0x30ee97c478d8a845d0d6ec0c447650ee6161817199f3d5c2a84f8a033b960b2",
-  "0x13b09830d3bef686db99c570df79ddfd813502cfa1d7c640ed871a0de6637ba",
-  "0x1e8f2ef3d53eb800432031fc3000b69b5132a2c4c0df821c20173ac8a91de6d",
-  "0x2523ffb01dad3d8803a4bae7e2990d74d92b201f5425f921cec2ccb92ee6f18",
-  "0xf6413b17b8ae645ce13e635df7dcb573086ff9558d8d9d7d2e343a250e4885",
-  "0x765e5f1ae2f6d796f359ea7c47c88387046d371aef1f02337921fe326fbacc2",
-  "0x21466caac9f9e2605100c76a0ab15e2cbc72942b876f4665fa490dd83f7a00f",
-  "0x28d165b48e0b523044bd790bae4cc7096d8747565b1d58691683662e12b6402",
-  "0x7297a3b129e3d3101bb9956338c9c64b7055556146c7ff2497caad4a831c11d",
-  "0x8d9da3628af39d3bb9a2155b4eb63af7791da1fd3864f93bb57130fa2b59d2",
-  "0x4dc2ebca0fa6de9e7f67e821e425a0f9a53cb300cc44cbd93d69c307ac536eb",
-  "0x6ea69b53281261da1cfde205a5e8c02db26bfa51df77419fef71014c53e03f1",
-  "0x3709159b0a92aefacba33401098c5b3c09240b67dd5c1d5ee9a778e6205028b",
-  "0x3ea725fcff7ddaffbdc1134be5ae164bcbf8155135de1d8663c5efb106c9b08",
-  "0x3135b4bb94b1b7c0d7a7074291d0f1d92ab57d1f61bddd0b7195f70207752ae",
-  "0x1eb1e028ffa31652cecb7d18311aac1a281610a535f20e7a3a82c13db226b0c",
-  "0x4f57bf04228bf2d2db2e0a8573c6594558a5f0b487c1229b5fe7723473afa58",
-  "0x2f24013ebb0911d5f27a2a2c3d07b8c73b2f36239d8325187ed25db1ce794b5",
-  "0x10e3a8574c22d19951d8e8715cc98a26b9ed9fe39f8e002cc82f49c189ea56c",
-  "0x6f823719688a9c3e54f7528d4e2a464773170365c2e8b97d699aa2cc519875e",
-  "0x669774f839f462c3267b1d28e9c31264e1436a48033ee0f71e77bef153cfb81",
-  "0x6ad46e2c0b5f52412560bb12ad77e2958d53e5f30fc5bc995101dd700e22597",
-  "0xe07332a9bf6f9d3ced6f3f1cb60607d01fba1f6be9da1b3650c58c8d683e06",
-  "0x3c320f3726b90a8e21d200eeb956f863c59c9d3744e5f5d9280c648225c7775",
-  "0xdc68dd15efb6b43a02a8679e4a21a9b8195cd44e6134d5034131520c55dbfa",
-  "0x594aa6e8d8d5ea9ea4a86681222f65975ed38c4e0f20e8493c0ad9887b3c3f4"
+  "0xff887caabf4c6d059cb674de16c3daef88e35256fb495201281f2315b1aae9",
```

Comment:
why did this change

Reply:
these were accounts on the old staging appchain

Reply:
why do we need them committed to the repo?

Reply:
i think we can remove if it's duplicated yes
unless we have a way to inline them in the deployment config